addons/website/static/tests/tours/snippet_table_of_content.js

```diff
@@ -2,6 +2,24 @@
 
 import wTourUtils from 'website.tour_utils';
 
+const scrollToHeading = function (position) {
+    return {
+        content: `Scroll to h1 number ${position}`,
+        trigger: `iframe h1`,
+        run: () => {
+            const iframeEl = document.querySelector('iframe');
+            iframeEl.contentDocument.querySelectorAll('h1')[position].scrollIntoView(true);
+        },
```

Comment:
Maybe:
```suggestion
        trigger: `iframe h1:nth(${position + 1})`,
        run: function () {
            this.$anchor[0].scrollIntoView(true);
        },
```
(/!\ use of `function` required to access `this.$anchor`).